integreat_compass/cms/templates/overlay/offer-detail-overlay.html

```diff
@@ -82,6 +82,81 @@ <h3 class="text-xl mt-4 font-semibold">{% translate "More details" %}</h3>
+                            <div id="comment-container-{{ offer.pk }}">
+                                {% for comment in offer.comments %}
+                                    <div class="mt-4">
+                                        <h4 class="font-extrabold">
+                                            {{ comment.creator.display_name }}
+                                            {% if offer.public_version != comment.offer_version %}
+                                                <div class="inline align-text-top text-sm text-gray-500 ml-2">
+                                                    <i icon-name="alert-triangle" class="h-4 w-4">
+                                                    </i> {% translate "Offer details have changed since this comment was added" %}
+                                                </div>
+                                            {% endif %}
+                                        </h4>
+                                        <p>{{ comment.comment }}</p>
+                                        <p class="text-sm text-gray-500">{{ comment.date|date:"d. M Y" }}</p>
+                                    </div>
+                                {% endfor %}
+                            </div>
+                        </div>
+                    </div>
+                    {% if offer.public_version.documents.first %}
+                        <div class="shrink-0 grow">
+                            <div class="mb-4">
+                                <h3 class="inline-block pb-4 pr-8 xl:pt-0 border-b-2 rounded-t-lg">{% translate "Documents" %}</h3>
+                                <ul>
+                                    {% for document in offer.public_version.documents.all %}
+                                        <li class="flex mt-4">
+                                            <div class="flex items-center">
+                                                {% if document.file_type == "application/pdf" %}
+                                                    <i icon-name="file-text"></i>
+                                                {% else %}
+                                                    <i icon-name="file-image"></i>
+                                                {% endif %}
+                                            </div>
+                                            <div class="ml-4">
+                                                <a href="{{ document.file.url }}" target="_blank" class="font-bold">{{ document.name }}</a>
+                                                <p>{{ offer.public_version.created_at }}</p>
+                                            </div>
+                                        </li>
+                                    {% endfor %}
+                                </ul>
+                            </div>
+                        </div>
+                    {% endif %}
+                </div>
```

Comment:
```suggestion
                </div>
         {% endif %}
```
Of course only, if you choose to follow my suggestion above